src/core/dbus-manager.c

```diff
@@ -1409,16 +1409,39 @@ static int dump_impl(
 
         _cleanup_free_ char *dump = NULL;
         Manager *m = ASSERT_PTR(userdata);
+        bool check_ratelimit = false;
         int r;
 
         assert(message);
 
-        /* Anyone can call this method */
-
+        /* 'status' access is the bare minimum always needed for this */
         r = mac_selinux_access_check(message, "status", error);
         if (r < 0)
                 return r;
 
+        /* Reload/reexec are also slow but non-destructive, use the same access check for dumps which, given
+         * the large amount of data to fetch, can stall for quite some time. If the caller is not privileged
+         * use rate limiting. */
+        r = mac_selinux_access_check(message, "reload", error);
```

Comment:
this "reload" thing is really confusing. And the comment doesn't really explain anything, it just looks like a copy-paste mistake?

Reply:
i.e. say clearly *what* you do here, and then *why* in the comment. You start out with some auxiliary fact that looks very misplaced.

Reply:
It's an attempt at explaining why the policy is checked against 'reload' rather than anything else. I'll add that we cannot add a new one, so have to use an existing one and this is the closest that I could find.

Reply:
expanded the comment, hopefully it makes more sense now